### PR TITLE
Added diversity statement.

### PR DESCRIPTION
**Pull request by freakboy3742**
Added a diversity statement, as ratified by the DSF board and the DSF Membership. 

A blog post announcing this change has been prepared, and is sitting on the production site awaiting publication.

### CODE REVIEW

File: djangoproject/templates/diversity/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{% extends "diversity/base.html" %}
+
+{% block title %}Django Community Diversity Statement{% endblock %}
+
+{% block content %}
+
+<h1>Django Community Diversity Statement</h1>
+
+{% for message in messages %}
```
Comment (jezdez) [inline djangoproject/templates/diversity/index.html]: Why are those messages needed?

Comment (jezdez) [inline djangoproject/templates/diversity/index.html]: Could be that you copied it from the code of conduct, but I think that's a bug there, too.

File: djangoproject/templates/diversity/base.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+{% block extrahead %}
+  {{ block.super }}
+  <style type="text/css">
+    h3 a.plink { display: none; }
+    h3:hover a.plink { display: inline; }
+    p.message {
+      background-color: #ffe761;
+      padding: 8px;
+    }
+  </style>
```
Comment (jezdez) [inline djangoproject/templates/diversity/base.html]: Can you elaborate what those styles are for?

File: djangoproject/templates/diversity/index.html (hunk 1 above)
Comment (freakboy3742) [inline djangoproject/templates/diversity/index.html]: I just copied the pages from the Code of Conduct. However, I agree there probably isn't much use for a messages block there.

File: djangoproject/templates/diversity/base.html (hunk 2 above)
Comment (freakboy3742) [inline djangoproject/templates/diversity/base.html]: Again - a copy from the code of conduct; I don't think they are needed.

**Comment by jezdez** [comment]
w00t, thanks @freakboy3742!

**Pull request merged**